src/wallet/node_rpc_proxy.cpp

```diff
@@ -85,6 +86,15 @@ void NodeRPCProxy::invalidate()
   m_rpc_payment_cookie = 0;
 }
 
+boost::optional<std::string> NodeRPCProxy::get_busy_syncing(bool &busy_syncing)
+{
+  auto res = get_info();
+  if (res)
+    return res;
+  busy_syncing = m_busy_syncing;
+  return boost::optional<std::string>();
+}
```

Comment:
This will actually check every 30 seconds minimum, which may not be quite what you want here. Is this busy flag going to be checked very often ?

Reply:
> Is this busy flag going to be checked very often ?

On every refresh